extra_docs/relations/README.md

```diff
@@ -0,0 +1,163 @@
+# Relations
+All project resources can be related to each other and SDK supports inserting, fetching and attaching/detaching related 
+records.
+Let's create a simple schema which will be used for all further examples:
+
+![Relations schema](https://jexia.github.io/jexia-sdk-js/assets/relations-schema.png) 
```

Comment:
I've got 404 for image, not sure if it is will be the same for final document

Reply:
It is going to be there when documentation will have been published